solution.txt

```diff
@@ -108,7 +104,8 @@ Exercise 10:
 
 // Complete Exercise 10 below...
 
-let hasSoup = foods.includes('soup')
+var hasSoup = 
+foods.indexOf('soup')===-1
```

Comment:
nice use of a boolean expression here! it' was a little unclear to me at first, though, because you split the variable assignment right after the assignment. you can keep this assignment to one line.